client/README.adoc

```diff
@@ -1,7 +1,4 @@
-= hawkular-btm
-:source-language: java
-
-== Client
+= Client
 
 The client folder contains components to be used instrument the client environment. Therefore we need to
```

Comment:
There are two places in this text where `org.hawkular.btm` is mentioned - could you update these aswell.